Backend/utils/ResponseCreation.py

```diff
@@ -0,0 +1,45 @@
+import json
+from flask import Response
+
+
+class ControllerResponse:
+  '''
+  An object to handle managment of
+  the JSON response that we want REST calls
+  to return. 
+  '''
+  def __init__(self):
+    self.errors = []
+    self.payload = {}
+    self.hasError = False
```

Comment:
The flag doesn't have to be there but in the future if we want to add in errors that don't stop us, imagine something along the lines of a warning, it will help us to.